core/src/main/java/org/elasticsearch/search/aggregations/metrics/percentiles/PercentilesBuilder.java

```diff
@@ -30,12 +31,14 @@
 
     private double[] percentiles;
     private Double compression;
+    private PercentilesMethod method;
+    private Integer numberOfSignificantValueDigits;
 
     /**
      * Sole constructor.
      */
     public PercentilesBuilder(String name) {
-        super(name, InternalPercentiles.TYPE.name());
+        super(name, InternalTDigestPercentiles.TYPE.name());
```

Comment:
Missed that this was here and needed sorting. I can put a String constant in one of the common classes for agg type?